editor/src/clj/editor/particle_lib.clj

```diff
@@ -133,7 +117,7 @@
     {:context context
      :prototype prototype
      :instances instances
-     :raw-vbufs (make-raw-vbufs (:emitters prototype-msg))
+     :raw-vbufs []
```

Comment:
These are now created on the fly, since we don't actually know what to create before it's used. As in, the size of the vbuffer depend on the size of a vertex now instead of a fixed vertex size, as it was before.